build.gradle

```diff
@@ -51,7 +49,7 @@ tasks.withType(JavaCompile).all {
 publishing {
     publications {
         maven(MavenPublication) {
-            groupId = 'org.checkerframework'
+            groupId = 'io.github.eisop'
```

Comment:
Like above, there should be a note here that this needs to be changed to something under the template users control.